src/sparseml/transformers/export.py

```diff
@@ -126,13 +131,29 @@ def load_task_model(task: str, model_path: str, config: Any) -> Module:
     raise ValueError(f"unrecognized task given of {task}")
 
 
+def load_task_dataset(task: str, tokenizer, data_args: Optional[Dict[str, Any]]):
+    """
+    :param task: the task a dataset being loaded for
+    :param tokenizer: the tokenizer to use for the dataset
+    :param data_args: additional data args used to create a `DataTrainingArguments`
+        instance for fetching the dataset
+    """
+    # TODO: fill out the function for
+    #  1) mlm
+    #  2) question-answering
+    #  3) text-classification
+    #  4) token-classification
+
+
 def export_transformer_to_onnx(
     task: str,
     model_path: str,
     sequence_length: int = 384,
     convert_qat: bool = True,
     finetuning_task: Optional[str] = None,
     onnx_file_name: str = MODEL_ONNX_NAME,
+    num_export_samples: int = 0,
+    dataset_name: Optional[str] = None,
```

Comment:
need to consider making this a dict of `DataArgs` as there may be more script specific arguments to load datasets (especially around custom datasets). can import the data arg classes by script specifically for parsing